artifactory/commands/transferfiles/fulltransfer.go

```diff
@@ -58,10 +59,13 @@ func (m *fullTransferPhase) phaseStarted() error {
 }
 
 func (m *fullTransferPhase) phaseDone() error {
-	err := setRepoFullTransferCompleted(m.repoKey)
-	if err != nil {
-		return err
+	// If the phase stopped gracefully (with ctrl+c), don't mark the phase as completed
```

Comment:
Since I think we should also add a `--stop` option for the `transfer-files` command (for cases when the command is ran a background process) I suggest removing the `(with ctrl+c)` from thre comment, since it'll soon not be the only method to stop the process.

The same goes for `filesdiff.go`.